e2e/scenario/validators.go

```diff
@@ -23,17 +23,23 @@ func DirectoryValidator(path string, files []string) *LiveVMValidator {
 	}
 }
 
-func SysctlConfigValidator(customSysctls map[string]int) *LiveVMValidator {
+func SysctlConfigValidator[V comparable](customSysctls map[string]V) *LiveVMValidator {
+	keysToCheck := make([]string, len(customSysctls))
+	for k, _ := range customSysctls {
+		keysToCheck = append(keysToCheck, k)
+	}
+	// regex used in sed command to remove extra spaces between two numerical values, used to verify correct values for
+	// sysctls that have string values, e.g. net.ipv4.ip_local_port_range, which would be printed with extra spaces
 	return &LiveVMValidator{
 		Description: "assert sysctl settings",
-		Command:     "sysctl -a",
+		Command:     fmt.Sprintf("sysctl %s | sed -E 's/([0-9])\\s+([0-9])/\\1 \\2/g'", strings.Join(keysToCheck, " ")),
```

Comment:
very annoying that we need this lol, though looks we do indeed if we use `sysctl`

Reply:
ugh, the same issue seems to occur when `cat`ing out the files themselves as well...

Reply:
I wonder why that happens, couldn't find an answer

Reply:
`tr -s ' '` (squashes whitespace) may help

`cat -A` if that doesn't work, to check for special chars before trimming.